### ISSUE TITLE
[a11y] "Tab sent from Firefox on <device name>" toast is not announced

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2929]
## Steps to reproduce
- enable voiceview
- sign in to FxA
- Send tab to device

### Expected behavior
Toast is announced, "Tab sent from Firefox on mcomella's mac", then page contents are announced.

### Actual behavior
Nothing is announced until page has sufficiently loaded. Because we don't announce page loading states, it's unclear they're even receiving a tab and could users.

### Device information
* Fire TV device: ?
* Latest affected Firefox version: v4.5+ master 2cc87063e1808eaaa291b818cb22c952c9469607
* Earliest affected Firefox version: ?
* Turbo Mode: ?

**Comment by mcomella** [comment]
P3: this is most concerning when there is someone else sending tabs to the device, which is less likely. With a single user, they'll probably remember they sent these tabs.
